### DISCUSSION TITLE
Multi-tenancy deployment

### DISCUSSION THREAD

**Discussion by Hamiltonspinola** [filamentphp/filament#13115] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Hey guys!

I need to implement a multi-tenancy application. But I'm having trouble displaying the correct items in the admin panel. Does anyone have any examples?

The context would be:

The system registers several companies. Within each company there must be employees with different levels of access (so far I have managed to implement it.) I need help so that a user from company A does not see data from company B.

**Comment by dissto** [comment]
Well, it would help if you'd share you current code though. (You can simplify it though)
